lib/rex/post/meterpreter/extensions/stdapi/fs/dir.rb

```diff
@@ -194,9 +195,9 @@ def Dir.unlink(path)
   # Downloads the contents of a remote directory a
   # local directory, optionally in a recursive fashion.
   #
-  def Dir.download(dst, src, recursive = false, force = true, &stat)
+  def Dir.download(dst, src, recursive = false, force = true, glob = nil, &stat)
```

Comment:
Great point. Most uses of download only set the first two arguments and rely on defaults for the rest, so the do not need updates. ui/console/command_dispatcher/extapi/clipboard.rb looks like it needs to be updated though. Why it passes 'true' as the fourth argument is strange to me.